sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -1,15 +1,39 @@
 
 * [Overview](#overview-of-sem)
-* [Syntax](#syntax)
-* [Operations](#operations)
-* [Resource types](#resource-types)
+  * [Syntax](#syntax)
+  * [Operations](#operations)
+  * [Resource types](#resource-types)
+    * [Dashboards](#dashboards)
+    * [Jobs](#jobs)
+    * [Notifications](#notifications)
+    * [Projects](#projects)
+    * [Secrets](#secrets)
 * [Working with organizations](#working-with-organizations)
+  * [sem connect](#sem-connect)
+  * [sem context](#sem-context)
 * [Working with resources](#working-with-resources)
-* [Adding one or more files in a new secret](#adding-one-or-more-files-in-a-new-secret)
+  * [sem create](#sem-create)
+  * [sem edit](#sem-edit)
+  * [sem get](#sem-get)
+  * [sem apply](#sem-apply)
+  * [sem delete](#sem-delete)
 * [Working with jobs](#working-with-jobs)
-* [The sem stop command](#the-sem-stop-command)
+  * [Creating one-off jobs](#creating-one-off-jobs)
+  * [sem attach](#sem-attach)
+  * [sem logs](#sem-logs)
+  * [sem port-forward](#sem-port-forward)
+  * [sem debug for jobs](#sem-debug-for-jobs)
+  * [On demand job creation](#on-demand-job-creation)
+  * [sem stop](#sem-stop)
```

Comment:
@markoa this last one doesn't follow the title naming pattern of `sem <x> for jobs`.